### ISSUE TITLE
❌ 🛟 The Ken Burns effect stopped working after update

### ISSUE THREAD

**Issue by Chlitzxer** [elementor/elementor#36557]
### Description

Hello! After the latest update to 3.33.0, I noticed that the Ken Burns effect on my home page stopped working on mobile devices. It works on desktop. I am using the Astra free theme and the free version of Elementor. Everything is updated to the latest version. All caches are cleared. I reverted to the previous version 3.32.4 and it worked again. Could you tell me where the problem might be or which of the new settings I should disable so that I don’t have to skip plugin updates just because of this.

Issue still present on Elementor 4.1.5 and Elementor Pro 4.1.3

To make it work you have to rollback to v3.32.5 this has been opened because an elementor support said to open the case here instead mentioned from https://wordpress.org/support/topic/the-ken-burns-effect-stopped-working-after-update/

He mentioned that you guys are aware but this was reported months ago and we don't see any fix yet.

### Steps to reproduce

1. Go to dashboard
2. Create a page.
3. Edit with Elementor
4. Create slideshow using latest version
5. Apply Ken Burn effects

### Expected behavior

Ken Burns should be visible using latest version and should not require to rollback elementor versions to make it work

### Elementor System Info

```txt
Latest Wordpress
Latest Elementor
```

### Agreement

- [ ] I confirm I have read and followed all the guidelines and instructions outlined in the Elementor Bug Report form.
- [x] I agree that my issue may be closed without further action if it doesn't meet all the requirements outlined in the Elementor Bug Report form.

**Comment by nicholaszein** [comment]
Hello. 

Your issue is missing the **required** [Elementor System Info](https://elemn.to/help-system-info) as outlined in both our [Contribution Guidelines for Reporting Bugs](https://elemn.to/gh-contributing-bug-reports) and the [Bug Report Form](https://elemn.to/gh-new-bug-report) you used to submit this issue. 

Without it, we cannot validate it, nor escalate your issue.

For this reason, I'm closing it as incomplete.

On a second note, make sure you have the latest version of Elementor installed, we're in version 4.1.4, and 4.1.3 for both Editor Core and Pro, and that Animation or Motion Effects are not disabled on your Operating System Accessibility Settings, as Elementor will not animate anything in accordance to accessibility laws. 

If you think this is still a bug, feel free to open a new issue with all the required information. We won't accept incomplete issues.
